package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ethos-design-system",
-  "version": "1.1.34",
+  "version": "1.1.35",
```

Comment:
Atm, I have to bump the version tag to get this to pass CI/CD (even though intuitively it feels  like I should be going back to a smaller number like 1.1.31 since I'm reverting stuff.

TODO: I think we should back out `ci/travis.spec.js` and rethink how we're doing all this cc / @zach-ethos @alexthecarp

Reply:
Yeah, I removed that spec to try to get EDS back to a known stable version which I believe is 1.1.31